index.js

```diff
@@ -25,6 +25,9 @@ function getTransformStream() {
       // See https://github.com/getsentry/sentry-javascript/issues/1964#issuecomment-688482615
       // 6 is enough to serialize the deepest property across all GitHub Event payloads
       normalizeDepth: 6,
+      environment: process.env.SENTRY_ENV,
+      debug: process.env.SENTRY_DEBUG === "true",
+      release: process.env.SENTRY_RELEASE,
```

Comment:
could you add tests for the additional configuration? Compare https://github.com/probot/pino/blob/03f0cf043bb3e29f4f0e32e79d7321dfb4dc850e/test/cli.test.js#L80-L136